c/openbsd-etherip/README

```diff
@@ -0,0 +1,8 @@
+Contributed by: Moritz Buhl
+
+Taken from the OpenBSD 6.2 src checkout https://www.openbsd.org/
+Copyright and license are mentioned in each file of the source directory.
+
+The create.sh file shows how to create the if_etherip_*.i files.
+To create the erreneous file, the patchfile in the sources directory needs
+to be reverted. (patch -R sys/if_etherip.c ./007_etherip.patch.sig)
```

Comment:
Isn't this done by the Makefile, as well? If it is, I would remove this sentence or rewrite it, because it implies that users have to do this to create the erroneous file.
This directory may also get more tasks, so I would reference the exact filename that you are talking about when saying `erroneous file`

How about something like:
```
The file `./007_etherip.patch.sig` contains the changeset that introduces the bug in $FILENAME_CORR (resulting file is $FILENAME_ERR)
```

Reply:
Removing that comment makes sense. Especially as the build is working without it.

> This directory may also get more tasks, so I would reference the exact filename that you are talking about when saying erroneous file

Then I would suggest naming the directory `openbsd-6.2` which also fits better with the other directory names e.g. `busybox-1.22.0` and `ldv-linux-3.14`. I just pushed a change.